### ISSUE TITLE
If failed to send encrypted email, send unencrypted error mail

### ISSUE THREAD

**Issue by MichaelDwucet** [MISP/MISP#3308]
On our MISP platform we forced in the MISP settings that outgoing emails have to be encrypted.

We now have a bunch of users, who's PGP keys have expired. Now I see regularly the following error messages in the logs:

Error message:
Email to "email address", titled "subject" failed. Reason: the message could not be encrypted because the provided key is either expired or cannot be used for encryption. 

So the users do not get any emails from the MISP platform and did not know about this.

Feature request:
If outgoing email could not be encrypted, have an option to send the user an unencrpyted error mail.

Example text:
Dear $user,
the message titled "subject" could not be sent to your account, because the provided key is either expired or cannot be used for encryption. 

Please check your PGP / S/MIME Key in your MISP profile and update it, if it has expired.

$Admin

**Comment by iglocska** [comment]
Good idea. We have to be careful not to spam users with these reminders though, so perhaps the best strategy would be to add a pgp warning lock on the user profile - send a warning, lock it and keep it locked until the user logs in again.

**Comment by adulau** [comment]
@iglocska The issue. It's a lot of users don't log very often because they rely on email notification, API or sync. The email replacement idea of @MichaelDwucet seems an acceptable solution as the spam number will be the number of potential notifications.

**Comment by iglocska** [comment]
Ok could do. Worst case we get complaints, in which case we can still rework the idea.
